app/constants/view.js

```diff
@@ -118,7 +118,7 @@ export default {
     ...ViewTypes,
     RequiredServer,
     POST_VISIBILITY_CHUNK_SIZE: 60,
-    CRT_CHUNK_SIZE: 60,
+    CRT_CHUNK_SIZE: 30,
```

Comment:
Why 30 instead of 60 ?

Reply:
Sorry this was included in this PR. Web app has implemented infinite scrolling and we had a discussion about number of threads to load and arrived at this number.